pyaerocom/aeroval/obsentry.py

```diff
@@ -101,22 +197,20 @@ def check_add_obs(self):
                 )
```

Comment:
Yes, this is leftover from the old "validators". The idea was that if it does not exist, it will crash at this step, which will tell a user that if they provided `obs_aux_requires` then they need to explcitly provide `obs_type`, and that if `obs_type` is not "ungridded", that currently it is not supported. We could add `obs_type` as an attribute with a default of "ungridded", but I'm concerned that people might forget to set it, and it could lead to unintended behavior. What do you think?

Reply:
Can we have `obs_type` default to `None` and check that it is None instead? Then the user has to assign it and it doesn't appear like a bug?

Reply:
Sure, added.